docs/src/cookbooks/cookbook.md

```diff
@@ -897,6 +897,31 @@ compile(new TooWideOrNarrowUInt(8, 2))
 compile(new TooWideOrNarrowUInt(8, 4))
 ```
 
+## How can I use Verilog "case equality" operators in Chisel?
+
+Verilog has "case equality" (`===`) and inequality (`!==`) operators.
+They are typically used to ignore unknown (`X`) values in assertions.
+
+Chisel does not support SystemVerilog `X` directly, but it is possible to check if a value is `X` with `chisel3.util.circt.isX`.
+`isX` is commonly used to guard assertions against `X` which gives similar behavior to Verilog case equality.
+
+```scala mdoc:silent:reset
+import chisel3._
+import chisel3.util.circt.IsX
+
+class XSafeAssert extends Module {
+  val in = IO(Input(UInt(8.W)))
+
+  // Assert that in is never zero, but also guard against X
```

Comment:
```suggestion
  // Assert that `in` is never zero, but also allow X
```

Reply:
I phrased it differently, what do you think?